aries-site/src/examples/templates/popover/PopoverSimpleExample.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useRef } from 'react';
+
+import { Popover } from 'aries-core';
+import { Button, Box, Paragraph, Text } from 'grommet';
+import { CircleInformation } from 'grommet-icons';
+
+export const PopoverSimpleExample = () => {
+  const [showPopover, setShowPopover] = useState(false);
+  const targetRef = useRef();
+
+  const handleButtonClick = () => {
+    setShowPopover(prev => !prev);
+  };
+
+  const handleClosePopover = () => {
+    setShowPopover(false);
+  };
+
+  return (
+    <Box direction="row" justify="center" align="center">
+      <Text>Click for Popover.</Text>
+      <Button
+        align="center"
+        justify="start"
+        icon={<CircleInformation />}
+        onClick={handleButtonClick}
+        ref={targetRef}
+        aria-expanded={showPopover}
```

Comment:
@KennyAtHPE @julauxen @britt6612 @taysea @jcfilben --- Question: should button receive active/selected state while the popover is open?

```suggestion
        active={showPopover}
        aria-expanded={showPopover}
```

Reply:
I already discussed this with Matt and Oliver but I am dropping in the rationale for visibility...

@halocline So originally we did consider this but I thought it'd be best if we mimicked the same behavior of our other components using drops. For example, all variants of the menu component (with text + border, icon-only, etc.) do not have an active/selected state when it is opened. To maintain consistency, I'd suggest keeping the same behavior for Popover triggers. 

![Screenshot 2024-10-17 at 9 55 03 AM](https://github.com/user-attachments/assets/aea90266-190b-45e8-9c9d-13aea228cd0f)

Furthermore, I don't believe this additional affordance is needed. If a user is intentionally clicking on these triggers, then they would be aware that the Popover is related to that trigger. It isn't a random action. I would also assume that this was the same rationale for not including an active/selected state within menu when originally designed.